d3-selection/index.d.ts

```diff
@@ -845,7 +878,7 @@ interface Selection<GElement extends BaseType, Datum, PElement extends BaseType,
     /**
      * Return the first (non-null) element in this selection. If the selection is empty, returns null.
      */
-    node(): GElement | null;
+    node(): GElement;
```

Comment:
Not at all, love the peer review. What I was getting at with the bikeshedding, was my reluctance  to put the `| null` back onto `node()`. I will let you know by @-mention when this PR and its companion are ready for a (fingers crossed) final review before merge. T